### DISCUSSION TITLE
Please make the color picker smaller

### DISCUSSION THREAD

**Discussion by Shubh-Khandelwal** [ScratchAddons/ScratchAddons#2603] [Ideas]
Please make the color picker smaller because when I open it with zoom size 100, it goes to little bit of top right and I am not able to use it properly.

**Comment by TheColaber** [comment]
Do you mean when opening it in the popup? Remember that the popup should never be used for editing addon settings. It should only be used to quickly turn on or off running addons.

**Reply by mxmou** [reply]
Editing settings of currently running addons is also an intended use of the popup. https://github.com/ScratchAddons/ScratchAddons/discussions/1663#discussioncomment-406252
